src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -91,12 +91,14 @@ public function reverseTransform($value)
             throw new \UnexpectedValueException(sprintf('Value should be array, %s given.', \gettype($value)));
         }
 
-        foreach ($value as $key => $id) {
-            if ('_labels' === $key) {
-                continue;
-            }
+        $ids = array_filter($value, static function ($key): bool {
+            return '_labels' !== $key;
+        }, ARRAY_FILTER_USE_KEY);
+
+        $singleIdentifierFieldName = $this->modelManager->getIdentifierFieldNames($this->className)[0];
```

Comment:
I think this wont be correct for composite keys

Reply:
I agree, but at the beginning I thought this was occurring with the previous implementation. Checking `EntityManager::find()` I see that isn't the case.
I guess we must build a custom query for composite keys, since if we call `findBy()` by each key, we have no guarantee about the keys' consistency.

Reply:
What I dont know if it is better to delegate that to the persistence bundles, because I dont know if the odm works the same way. Maybe it is better to introduce the findMultiple you mentioned in the issue